src/helper/core/goe-api.ts

```diff
@@ -34,3 +36,24 @@ export async function createImpl(env: NodeJS.ProcessEnv): Promise<Api> {
     }
   };
 }
+
+function getGasIncPct(): number {
+  const envKeys = Object.keys(process.env);
+
+  const patterns = [
+    /^GOE?_?GAS(_INC)?_?PCT$/i,   // GOE_GAS_INC_PCT, GOE_GAS_PCT, GAS_INC_PCT, GAS_PCT
+    /^GOE?_?INC_?GAS$/i,           // GOE_INC_GAS, INC_GAS
+  ];
+
+  for (const pattern of patterns) {
+    const key = envKeys.find(k => pattern.test(k));
+    if (key) {
+      const value = process.env[key];
+      if (value) {
+        const n = Number(value);
+        if (!isNaN(n)) return n;
+      }
+    }
+  }
+  return 0;
```

Comment:
The default value could be 10% so that user can have more possibility to get quick confirmation?